common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -693,56 +699,57 @@ def get_problem_html(self, encapsulate=True, demand_hint_text=None, hint_index=0
 
         return html
 
-    def _get_answer_notification(self):
+    def _get_answer_notification(self, render_notifications=False):
```

Comment:
Let's not make this an optional variable. It's called in exactly one place, and it would be a bug if render_notifications is not specified.